static/js/browser2/observation_chart.tsx

```diff
@@ -81,159 +76,61 @@ export class ObservationChart extends React.Component<
 
   private plot(): void {
     const values = this.props.sourceSeries.val;
-    if (_.isEmpty(values)) {
-      return;
-    }
-    const plotData = [];
-    Object.keys(values).forEach((key) =>
-      plotData.push({
-        dateString: key,
-        time: Date.parse(key),
+    const data = [];
+    Object.keys(values).forEach((key) => {
+      data.push({
+        label: key,
         value: Number(values[key]),
-      })
+      });
+    });
+    const dataGroups = [new DataGroup("", data)];
+    const svgContainerId: string = "svg-container" + this.props.idx;
+    drawLineChart(
+      svgContainerId,
+      WIDTH,
+      HEIGHT,
+      dataGroups,
+      true,
```

Comment:
please add a comment that even though all dots are drawn, they are not displayed except on hover (and that is controlled via css).

Reply:
Actually, all the dots are displayed even when not hovering https://screenshot.googleplex.com/B8sxhGugVGBqPG2

Reply:
Ah.. we might want to have a limit for that, since it won't work out so well for dense timelines (like unemployment)

Reply:
oh, I see what you mean. I can look into only showing dots on hover when there is a dense timeline